src/parser/jobs/rpr/modules/OtherGauges.tsx

```diff
@@ -0,0 +1,142 @@
+	}))
+
+	private soulGaugeModifiers = new Map<Action['id'], GaugeModifier>([
+		// Builders - other than Design since it's a status
+		[this.data.actions.SLICE.id, {action: 10}],
+		[this.data.actions.WAXING_SLICE.id, {combo: 10}],
+		[this.data.actions.INFERNAL_SLICE.id, {combo: 10}],
+		[this.data.actions.SPINNING_SCYTHE.id, {action: 10}],
+		[this.data.actions.NIGHTMARE_SCYTHE.id, {combo: 10}],
+		[this.data.actions.SOUL_SLICE.id, {action: 50}],
+		[this.data.actions.SOUL_SCYTHE.id, {action: 50}],
+		// Spenders
+		[this.data.actions.BLOOD_STALK.id, {action: -50}],
+		[this.data.actions.GRIM_SWATHE.id, {action: -50}],
+		[this.data.actions.UNVEILED_GIBBET.id, {action: -50}],
+		[this.data.actions.UNVEILED_GALLOWS.id, {action: -50}],
+		[this.data.actions.GLUTTONY.id, {action: -50}],
+	])
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		const foeFilter = filter<Event>().actor(oneOf(this.actors.foes.map(a => a.id)))
+
+		// Soul Gauge
+		const soulActions = Array.from(this.soulGaugeModifiers.keys())
+		this.addEventHook(playerFilter.type(oneOf(['action', 'combo'])).action(oneOf(soulActions)), this.onSoulModifier)
+		this.addEventHook(foeFilter.type('death'), this.onFoeDeath)
+
+		// Shroud Gauge
+		this.addEventHook(playerFilter.type('action').action(this.data.matchActionId(SHROUD_ACTIONS)), this.onShroudModifier)
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onFoeDeath(event: Events['death']) {
+		// We effectively check if they had the status 100ms before death, since it drops with death
+		if (this.actors.get(event.actor).at(event.timestamp - 100).hasStatus(this.data.statuses.DEATHS_DESIGN.id)) {
+			this.soulGauge.generate(DESIGN_GAIN)
+		}
+	}
+
+	private onSoulModifier(event: Events['action' | 'combo']) {
+		const modifier = this.soulGaugeModifiers.get(event.action)
+
+		if (modifier != null) {
+			const amount = modifier[event.type] ?? 0
+			this.soulGauge.modify(amount)
+		}
+	}
+
+	private onShroudModifier(event: Events['action']) {
+		const action = this.data.getAction(event.action)
+		if (action == null) { return }
+
+		switch (action.id) {
+		case this.data.actions.ENSHROUD.id:
+			this.shroudGauge.spend(HIGH_SHROUD_MOD)
+			break
+
+		case this.data.actions.PLENTIFUL_HARVEST.id:
+			this.shroudGauge.generate(HIGH_SHROUD_MOD)
+			break
+
+		default:
+			this.shroudGauge.generate(BASE_SHROUD_MOD)
+		}
+	}
+
+	private onComplete() {
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SOUL_SLICE.icon,
+			content: <Trans id="rpr.gauge.soul.suggestions.overcap.content">
+					Avoid letting your Soul gauge overcap. The wasted resources cost you Shroud generation which you need for your highest damage skills.
+			</Trans>,
+			why: <Trans id="rpr.gauge.soul.suggestions.overcap.why">
+				{this.soulGauge.overCap} Souls lost to overcapping Soul gauge.
```

Comment:
nit: this value should be /10 given the existing copy, IMO.

Reply:
Might be a bit of a disconnect here. The job guide and game itself call each point of gauge "souls", even tho you realistically consume it in much larger blocks. The idea here is to show how much total gauge was overcapped, not individual actions. RPR only has one real use for Soul gauge tho, so maybe assuming that use is fine. Open to ideas, the current approach is more in alignment with what other jobs do but I'm not married to that.

Reply:
Ah, noted. Makes sense to me.